src/libraries/Microsoft.Extensions.Configuration.EnvironmentVariables/src/EnvironmentVariablesConfigurationProvider.cs

```diff
@@ -38,6 +38,13 @@ public class EnvironmentVariablesConfigurationProvider : ConfigurationProvider
         public override void Load() =>
             Load(Environment.GetEnvironmentVariables());
 
+        /// <summary>
+        /// Generates a string representing this provider name and relevant details.
+        /// </summary>
+        /// <returns> The configuration name. </returns>
+        public override string ToString()
```

Comment:
This PR didn't update the ref.  If someone derives from the unsealed EnvironmentVariablesConfigurationProvider and calls base.ToString, this ToString will be skipped.

Also, tests?

Reply:
How did the build pass when the ref wasn't updated 🧐

Reply:
How did this thing fail the build for whitespace but not this if it doesn't work 😄

Reply:
Overrides are not required to be on the ref as it is valid to promote an override to the base type at any time.

Reply:
Is there a scenario where I would want to override in the implementation but not the ref?

Reply:
I don't think so. Good observation, I'll consider it for the new API Compat.

Cc: @ericstj

Reply:
This is bug https://github.com/dotnet/arcade/issues/2570#issuecomment-702402341

Reply:
Good to know. I thought it was intentional, we actually are honoring that behavior on the new API Compat.

Reply:
It was intentional in APICompat originally.   APICompat tooling was originally created to help ensure that PCL was legitimate and make sure that it could expose reference assemblies that would be satisfied by Silverlight, Windows Phone, etc which in some cases were quite different.  We've since started using it for more things where we might want to have more strict requirements.